assets/javascripts/discourse/widgets/encrypt-time-bomb-counter.js.es6

```diff
@@ -0,0 +1,26 @@
+import { createWidget } from "discourse/widgets/widget";
+import { iconNode } from "discourse-common/lib/icon-library";
+
+createWidget("encrypt-time-bomb-counter", {
+  tagName: "div.time-bomb-counter",
+
+  init() {
+    setTimeout(() => {
+      this.scheduleRerender();
```

Comment:
This seems fishy to me. You do not clear the timeout if the widget is destroyed and it may be out of sync with the actual value if the first render happens right before the "formattedClock" string would change.